Formula/sparkctl.rb

```diff
@@ -0,0 +1,29 @@
+class Sparkctl < Formula
+  desc "CLI for interacting with spark on k8s"
+  homepage "https://github.com/GoogleCloudPlatform/spark-on-k8s-operator"
+  url "https://github.com/GoogleCloudPlatform/spark-on-k8s-operator.git",
+      tag:      "v1beta2-1.2.2-3.0.0",
+      revision: "2e8b733f5ad029ceb8a949b153a53c434b8fcfe1"
+  # the versioning scheme is apiversion-operatorVersion-sparkVersion.
+  # Homebrew will automatically detect the spark version by default
+  # instead pull the operator version which more closely matches to sparkctl
+  version "1.2.2"
+  license "Apache-2.0"
+  head "https://github.com/GoogleCloudPlatform/spark-on-k8s-operator.git"
+  depends_on "go@1.15" => :build
```

Comment:
```suggestion

  depends_on "go@1.15" => :build
```
Does this not support building with Go 16?

Reply:
current upstream uses go 15. it compiles with 16 but i haven't verified everything works. I can change it and do some testing.